### ISSUE TITLE
Discussion: What are some of the major issues surrounding Internet health and misinformation? 

### ISSUE THREAD

**Issue by secmorris** [nucleus-network/missioninfo#18]
Discuss issues, trends, problems you'd like to see discussed or tackled here.

**Comment by MkllTech** [comment]
The spread of misinformation on Social Media grows rampant every day, I'd like to see these platforms, or -- at least the people who use them; be able to identify when something trending is untrue, and not an actual news story.

We've seen this happen on Twitter before with the rise of political propaganda bots, something that is still going on in the world of Twitter today!

**Comment by secmorris** [comment]
@MkllTech Completely agree about having more info about when something untrue is trending, whether that’s tools that can help or just educating people to recognize when that’s happening. 

I’d love to put material or an activity together on bots! Let me know if you’d like to chat more about that!

**Comment by vschs007** [comment]
@MkllTech @secmorris I agree with you, apart from this there are automated bots, political parties funded IT firms for false information pushing to Internet.
There is issues with the users themselves as many people spread the false information without checking its validity just because it was from a source they usually believe to be a genuine one.
